src/app/controllers/admin-forms.server.controller.js

```diff
@@ -715,5 +715,44 @@ function makeModule(connection) {
         },
       )
     },
+
+    /**
+     * Transfer a form to another user
+     * @param  {Object} req - Express request object
+     * @param  {Object} res - Express response object
+     */
+    transferOwner: async function (req, res) {
+      const newOwnerEmail = req.body.email
+
+      // Check if new owner's email is specified
+      if (_.isEmpty(newOwnerEmail)) {
+        const errMsg = 'Error: email should exists in req body'
+        logger.error(
+          `formId="${req.form._id}", ip=${getRequestIp(
+            req,
+          )}, message="${errMsg}"`,
+        )
+        return res.status(HttpStatus.BAD_REQUEST).send({ message: errMsg })
+      }
```

Comment:
Can we move this validation to the route file and do it using `Joi`?

Reply:
I believe there should be a `email` type

Reply:
Okay